File: src/kyu_7/deodorant_evaporator/js/evaporator.js
function evaporator(content, evap_per_day, threshold){
    let day = 0;
    let total = content;
    while (content >= total * (1 / 100) * threshold) {
        content -= content * (1 / 100) * evap_per_day;
        day++;
    }
    return day;
}
module.exports = evaporator;

File: src/kyu_7/deodorant_evaporator/java/Evaporator.java
package kyu_7.deodorant_evaporator.java;

public class Evaporator {

  public static int evaporator(double content, double evap_per_day, double threshold) {
    int day = 0;
    double total = content;
    while (content >= total * (1d / 100) * threshold) {
      content -= content * (1d / 100) * evap_per_day;
      day++;
    }
    return day;
  }
}

/*
10мл
10% от 10 мл = 1
10% порог = 1 -> распылитель не работает
на выходе 22 -> день, когда испаритель выйдет из строя


 */